private_sharing/models.py

```diff
@@ -526,10 +538,21 @@ def leave_project(self, remove_datafiles=False, done_by=None, erasure_requested=
             items.delete()
 
     def save(self, *args, **kwargs):
+        """
+        Oviride to record when a project is joined and authorized and generate a
```

Comment:
Typo here. :)

Also a general question -- maybe @gedankenstuecke has an opinion too -- would it be better to be recording all join/authorization history, as was done for the record of project approval (#948)?

Reply:
Mh... I think it wouldn't harm to store that data instead of losing it (even if I don't have a immediate use case in mind)!

Reply:
Went ahead and did that thing